apps/ledger-live-desktop/tests/specs/speculos/delete.account.spec.ts

```diff
@@ -5,28 +5,31 @@ import { Application } from "tests/page";
 
 const accounts: Account[] = [
   Account.BTC_1,
-  Account.tBTC_1,
   Account.ETH_1,
-  Account.tETH_1,
-  Account.sep_ETH_1,
   Account.SOL_1,
   Account.TRX_1,
   Account.DOT_1,
   Account.XRP_1,
+  Account.ADA_1,
+  Account.ALGO_1,
+  Account.XLM_1,
+  Account.BCH_1,
+  Account.ATOM_1,
+  Account.XTZ_1,
 ];
 
 for (const [i, account] of accounts.entries()) {
   test.describe.parallel("Accounts @smoke", () => {
     test.use({
-      userdata: "speculos",
+      userdata: "speculos-tests-app",
       testName: `deleteAccount_${account.currency.uiName}`,
       speculosCurrency: specs[account.currency.deviceLabel.replace(/ /g, "_")],
       speculosOffset: i,
     });
 
     //@TmsLink("B2CQA-320")
 
-    test(`[${account.currency.uiName}] Receive`, async ({ page }) => {
+    test(`[${account.currency.uiName}] Delete Account`, async ({ page }) => {
```

Comment:
on my side only 4 test passed and 8 failed with `{"stack":"Error: Date.parse is not a function\`
<img width="959" alt="image" src="https://github.com/LedgerHQ/ledger-live/assets/91890425/f948a510-0594-4d75-bca0-9686d6b7dce0">

Reply:
That's really weird, I just run the tests and everything is green 🤔 
<img width="1147" alt="Screenshot 2024-06-03 at 17 07 12" src="https://github.com/LedgerHQ/ledger-live/assets/162306106/3f814da4-de8c-457c-835c-184949d14df9">